openeo_processes_dask/process_implementations/cubes/_filter.py

```diff
@@ -88,3 +91,69 @@ def filter_bands(data: RasterCube, bands: list[str] = None) -> RasterCube:
             f"The provided bands: {bands} are not all available in the datacube. Please modify the bands parameter of filter_bands and choose among: {data[band_dim].values}."
         )
     return data
+
+
+def filter_bbox(data: RasterCube, extent: BoundingBox) -> RasterCube:
+    try:
+        input_crs = str(data.rio.crs)
+    except Exception as e:
+        raise Exception(f"Not possible to estimate the input data projection! {e}")
+    trasnformed_extent = reproject_bbox(extent, input_crs)
+
+    # Check if the coordinates are increasing or decreasing
+    if len(data.y) > 1:
```

Comment:
There should be an else statement to this, otherwise the slice might be undefined when it's called later